lib/project_types/extension/cli.rb

```diff
@@ -62,6 +62,12 @@ module Questions
   end
 
   module Features
+    module Runtimes
+      autoload :Admin, Project.project_filepath("features/runtimes/admin")
+      autoload :Default, Project.project_filepath("features/runtimes/default")
```

Comment:
Nit: After reading the rest of the code, I feel like `Base` would be a more accurate name than `Default`. WDYT?